.circleci/config.yml

```diff
@@ -461,9 +461,6 @@ jobs:
                 command: |
                   echo "PR Target branch: $PR_TARGET_BRANCH"
                   ./gradlew sdk:dependencyInsight --configuration releaseRuntimeClasspath --dependency com.mapbox.maps | grep -vzq 'FAILED'
-            - run:
```

Comment:
so, right now internal `build from source` should be required?
or maybe add extra upstream job that triggers only license check?

Reply:
I don't think its a `required` job, we will keep it sync, and just validate license file on everytime we . 
I guess no need to add extra license check, it will be triggered everytime we create a commit / merge to main (Similar to codegen-verify).

Reply:
yeah, but if somethings change in public PR, you just run license check as a part of `build from source` and it's not required job. Potentially you can merge it

Reply:
I thought about that to include in `build-from-source`, but as the name suggest it builds everything from source, and it doesn't look correct if we verify licenses in that job. 

> and it's not required job. Potentially you can merge it

Yes, potentially we should be able to merge imo. If there is some issue with the license file or let's say license-plugin, it shouldn't stop our development flow.

Reply:
I'd trigger a gl-native bot to validate the license on every PR, in case we accidentally introduce a new dependency while developing the public sdk.

Reply:
> accidentally introduce a new dependency

we don't generate new licenses from c++ code, we just include those from gl-native. so i don't think we can accidentally introduce new dependency from our side. 
one case i can think is if certain licenses are included in `internal` of gl-native but not present in release branch. I think gl-native should keep the internal licenses upto date with these inclusion.

Reply:
I mean the dependency we introduce from Android SDK, not related to gl-native.